sdks/go/pkg/beam/runners/prism/internal/worker/bundle.go

```diff
@@ -206,6 +206,20 @@ func (b *B) Cleanup(wk *W) {
 	wk.mu.Unlock()
 }
 
+func (b *B) Finalize(ctx context.Context, wk *W) (*fnpb.FinalizeBundleResponse, error) {
+	resp := wk.sendInstruction(ctx, &fnpb.InstructionRequest{
+		Request: &fnpb.InstructionRequest_FinalizeBundle{
+			FinalizeBundle: &fnpb.FinalizeBundleRequest{
+				InstructionId: b.InstID,
+			},
+		},
+	})
+	if resp.GetError() != "" {
+		return nil, fmt.Errorf("finalize[%v] error from SDK: %v", b.InstID, resp.GetError())
+	}
+	return resp.GetFinalizeBundle(), nil
```

Comment:
This is and will likely always be an empty message, and since this is internal code, I'd rather simply not return the empty message at this time, than speculate that we might want to return the message in the future if it gains fields.